x-pack/plugin/esql/qa/testFixtures/build.gradle

```diff
@@ -12,3 +12,21 @@ dependencies {
     implementation project(':server')
     api "net.sf.supercsv:super-csv:${versions.supercsv}"
 }
+
+/**
+ * Runs CSV Spec Tests data loader to load data to a running stand-alone instance
+ * Accepts an URL as first argument, eg. http://localhost:9200 or http://user:pass@localhost:9200
+ *
+ * eg.
+ * ./gradlew :x-pack:plugin:esql:qa:testFixtures:loadCsvSpecData --args="http://elastic-admin:elastic-password@localhost:9200"
+ *
+ * If no arguments are specified, the default URL is http://localhost:9200 without authentication.
+ * It also supports HTTPS.
+ */
+task loadCsvSpecData(type: JavaExec) {
```

Comment:
Adding this task to run `CsvTestsDataLoader.main` from CLI, mostly because I keep having classpath issues when I try to run it from IntelliJ as a normal `main()` method